codice/ddf
# Upgrade to Camel 3.14.0

#### What does this PR do?
Upgrades to Camel 3.14. Upgrading Camel also required moving from AWS Java SDK 1.11.x to AWS Java SDK 2.17.x.

#### Who is reviewing it? 


#### Select relevant component teams: 


#### Ask 2 committers to review/merge the PR and tag them here.

@ahoffer
@andrewzimmer
@ryeats

#### How should this be tested?

Verify CI build passes. Unzip, run, install the standard profile, ingest files, and query. Verify you can install the `catalog-metacard-backup-s3storage` feature.

#### Any background context you want to provide?

#### What are the relevant tickets?
Fixes: n/a

#### Screenshots


#### Checklist:
- [ n/a ] Documentation Updated
- [ n/a ] Update / Add Threat Dragon models
- [ n/a ] Update / Add Unit Tests
- [ n/a ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
